vm/block_test.go

```diff
@@ -0,0 +1,49 @@
+package vm
+
+import "testing"
+
+func TestBlockInitialize(t *testing.T) {
+	tests := []struct {
+		input    string
+		expected interface{}
+	}{
+		{`
+b = Block.new do
```

Comment:
I also implemented something like `Proc.new`. But I always don't think `Proc` is a good name, so I just call it `Block`.
And I **won't** support something like `proc`.